src/worldofzuul/Room.java

```diff
@@ -21,6 +27,16 @@
     private HashMap<String, Room> exits;
 
     /**
+     * The ArrayList keeps the items
+     */
+    private ArrayList<Item> items;
```

Comment:
Think an array would be better here.

Reply:
fixed in: 718f5ca2b3d2a167093b40a4fc2b4826fc510239